src/camvidlog2/queries.py

```
@@ -0,0 +1,68 @@
+    frame_count = 0
+    for i in embedding_group.items:
+        if isinstance(i, StringEmbedding):
+            nda_combined_list.append(nda_strings[string_count, :])
+            string_count += 1
+        elif isinstance(i, FrameEmbedding):
+            nda_combined_list.append(
+                frame_embedding_data.loc[(str(i.filepath), i.frame_no), :]
+            )
+            frame_count += 1
+        else:
+            raise RuntimeError("Unexpected item in embedding_group")
```

Ruff (0.11.9)

Prefer `TypeError` exception for invalid type

(TRY004)